projects/skia/build.sh

```diff
@@ -15,17 +15,45 @@
 #
 ################################################################################
 
+# Build SwiftShader
+cd SwiftShader
+export SWIFTSHADER_INCLUDE_PATH=$PWD/include
+rm -rf build
+mkdir build
+
+cd build
+if [ $SANITIZER == "profile" ]; then
+  cmake ..
+else
+  if [ $SANITIZER == "address" ]; then
+    CMAKE_SANITIZER="ASAN"
+  elif [ $SANITIZER == "memory" ]; then
+    CMAKE_SANITIZER="MSAN"
+  elif [ $SANITIZER == "undefined" ]; then
+    CMAKE_SANITIZER="UBSAN"
+  else
+    exit 1
+  fi
+  cmake .. -D$CMAKE_SANITIZER=1
+fi
+
+make -j
+cp libGLESv2.so libEGL.so $OUT
+export SWIFTSHADER_LIB_PATH=$OUT
+
+cd ../../skia
 # These are any clang warnings we need to silence.
 DISABLE="-Wno-zero-as-null-pointer-constant -Wno-unused-template
          -Wno-cast-qual -Wno-self-assign -Wno-return-std-move-in-c++11"
 # Disable UBSan vptr since target built with -fno-rtti.
-export CFLAGS="$CFLAGS $DISABLE -fno-sanitize=vptr"
-export CXXFLAGS="$CXXFLAGS $DISABLE -fno-sanitize=vptr"
+export CFLAGS="$CFLAGS $DISABLE -I$SWIFTSHADER_INCLUDE_PATH -DGR_EGL_TRY_GLES3_THEN_GLES2 -fno-sanitize=vptr"
+export CXXFLAGS="$CXXFLAGS $DISABLE -I$SWIFTSHADER_INCLUDE_PATH -DGR_EGL_TRY_GLES3_THEN_GLES2 -fno-sanitize=vptr"
+export LDFLAGS="-lFuzzingEngine $CXXFLAGS -L$SWIFTSHADER_LIB_PATH"
```

Comment:
Are we sure $LDFLAGS should have all of $CXXFLAGS?

Reply:
It definitely doesn't need all flags.
But I think this is the easiest way to get the subset of flags it does need since that subset will vary from build to build (eg: -fsanitize=address, -fsanitize=memory).
I believe we were basically doing the same thing before, except we didn't have an explicit LDFLAGS_ARRAY variable.